### DISCUSSION TITLE
Parallel installations

### DISCUSSION THREAD

**Discussion by AnjaliMamidi** [wixtoolset/issues#7046] [Questions]
Hello,

I'm trying to implement parallel installations for wix setups. But with the existing versions of wix, this functionality is not supported. Will this functionality be supported in future versions of wix? Or any guidance regarding parallel installations would be appreciated.

Thanks!!!

**Comment by robmen** [comment]
Are you talking about Burn or the Windows Installer?

* If Burn, that is owned by the WiX Toolset, but it does not prevent parallel installations by default.
* If the Window Installer, that is owned by Microsoft and it does block parallel installations.

I _think_ you want to follow up with Microsoft.

**Reply by AnjaliMamidi** [reply]
Thanks Robmen!!!!

Any insights or information you can provide on how we can achieve parallel installations using WiX Toolset.

Thanks in advance!!!!!
